services/rabbit/docker-compose.loadbalancer.yml.j2

```diff
@@ -23,6 +23,23 @@ services:
         reservations:
           cpus: "0.1"
           memory: "128M"
+      labels:
+        - traefik.enable=true
+        - traefik.swarm.network=${PUBLIC_NETWORK}
+        - traefik.http.services.rabbit_dashboard.loadbalancer.server.port=${RABBIT_MANAGEMENT_PORT}
+        - traefik.http.routers.rabbit_dashboard.rule=Host(`${MONITORING_DOMAIN}`) && PathPrefix(`/rabbit`)
+        - traefik.http.routers.rabbit_dashboard.entrypoints=https
+        - traefik.http.routers.rabbit_dashboard.tls=true
+        - traefik.http.middlewares.rabbit_dashboard_replace_regex.replacepathregex.regex=^/rabbit/(.*)$$
+        - traefik.http.middlewares.rabbit_dashboard_replace_regex.replacepathregex.replacement=/$${1}
+        - traefik.http.routers.rabbit_dashboard.middlewares=rabbit_dashboard_replace_regex@swarm, ops_gzip@swarm
```

Comment:
minor: I am reading some blog articles since some months that one should be careful with zipping, and not do it by default. we do it everywhere without checking. there might be quick-win performance gains if we remove it, who knows.... just dropping this here, not important for the rabbit per se